docs/content/docs/build/auth.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Authentication & authorization"
+menuPosition: 3
+editor: tylerbutler
+---
+
+Security is critical to modern web applications. Fluid Framework, as a part of your web application architecture, is an
+important piece of infrastructure to secure. Fluid Framework is a layered architecture, and auth-related concepts are
+primarily a concern of the *driver* layer (see [Architecture]({{< relref "architecture.md" >}})). This means that the
+specifics of authentication could differ based on the Fluid service and its driver.
+
+The information below is based on Azure Fluid Relay service (FRS) but also applies to
+[Tinylicious]({{< relref "tinylicious.md" >}}). Other Fluid services may differ.
+
+## Azure Fluid Relay service
+
+{{< include file="_includes/frs-onboarding.html" safeHTML=true >}}
+
+Each FRS tenant you create is assigned a *tenant ID* and its own unique *tenant secret key*.
+
+The secret key is a *shared secret*. Your app/service knows it, and FRS knows it. This means that you can sign data
+using that secret key, and FRS can verify that it is you who signed those requests because it also has that key.
```

Comment:
@tanviraumi to confirm if this is actually a *shared secret* or a *public/private key*. The main difference is if the secret can be used to both encrypt and decrypt the JWT.

Reply:
Great point.

Reply:
Tanvir confirmed it's a shared secret.